website/docs/d/pi_pvm_snapshots.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_pvm_snapshots"
+description: |-
+  Manages an pvm instance snapshots in the Power Virtual Server cloud.
+---
+
+# ibm_pi_pvm_snapshots
+Retrieve information about a Power Systems Virtual Server PVM instance snapshots. For more information, about Power Virtual Server PVM instance snapshots, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
```

Comment:
should I remove PVM in the first line?
Retrieve information about a Power Systems Virtual Server "PVM" instance snapshots.